lib/java_buildpack/container/tomee/tomee_instance.rb

```diff
@@ -34,6 +44,21 @@ def tomcat_7_compatible
         @version < TOMEE_7
       end
 
+      private
+
+      def web_inf_lib
+        if ear?
+          # copy additional libraries to tomee lib folder
+          tomcat_lib
```

Comment:
That tomee discussion thread is very interesting! your solution proposal sounds great @violetagg , 1 is a good intermediate step to 2. 

A potential issue from a user perspective would be on how to get the jars in the drivers folder but that can be handled later.